staging/cse/windows/provisioningscripts/securetlsbootstrap.ps1

```diff
@@ -0,0 +1,122 @@
+Param(
```

Comment:
Please also update README in staging/cse/windows and update the script to publish aks windows cse script package in ado repo

Reply:
fixed

Reply:
After it is merged but before you publish new cse script package, you need to update the script to publish aks windows cse script package in ado repo